skyvern/library/skyvern_browser_page.py

```
@@ -547,6 +548,116 @@ async def select_option(
+        """
+
+        files = files or ""
+
+        if ai == "fallback":
+            error_to_raise = None
+            if selector:
+                try:
+                    file_path = await download_file(files)
+                    locator = self._page.locator(selector)
+                    await locator.set_input_files(file_path)
+                except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)